DigitalLearningSolutions.Data/DataServices/SelfAssessmentDataService/SelfAssessmentSupervisorDataService.cs

```diff
@@ -234,25 +234,24 @@ int delegateUserId
                         Email,
                         ProfileImage,
                         IsFrameworkDeveloper
-                    FROM AdminUsers
-                    WHERE (
-                        (Supervisor = 1 OR NominatedSupervisor = 1) AND (Active = 1) AND (CategoryID = 0) AND (CentreID = @centreId)
-                        OR
-                        (Supervisor = 1 OR NominatedSupervisor = 1) AND (Active = 1) AND (CategoryID = (SELECT CategoryID FROM SelfAssessments WHERE (ID = @selfAssessmentId))) AND (CentreID = @centreId)
-                    )
+                        FROM AdminUsers
+                        WHERE 
+                        CentreID IN (SELECT CentreID FROM DelegateAccounts where UserID = @delegateUserId)

```

Comment:
I think we need some additional checks here

Reply:
This is fixed now. Please have a look